code/game/objects/items/storage/uplink_kits.dm

```diff
@@ -168,15 +168,16 @@
 			new /obj/item/assembly/signaler(src) // 0 tc
 			new /obj/item/storage/toolbox/syndicate(src) // 1 tc
 			new /obj/item/pen/edagger(src)
-			new /obj/item/gun/energy/wormhole_projector(src) //mooorttyyyy
+			new /obj/item/gun/energy/wormhole_projector(src)
 			new /obj/item/gun/energy/decloner/unrestricted(src)
 
 		if("bee")
 			new /obj/item/paper/fluff/bee_objectives(src) // 0 tc (motivation)
 			new /obj/item/clothing/suit/hooded/bee_costume(src) // 0 tc
 			new /obj/item/clothing/mask/rat/bee(src) // 0 tc
 			new /obj/item/storage/belt/fannypack/yellow(src) // 0 tc
-			new /obj/item/storage/box/syndie_kit/bee_grenades(src) // 15 tc
+			new /obj/item/grenade/spawnergrenade/buzzkill(src)
+			new /obj/item/grenade/spawnergrenade/buzzkill(src)
```

Comment:
You can probably safely remove the bee grenade box completely and please add comments comments that tell you the TC calculation, might sound inane, but it's honestly fairly important to keep such stuff maintained.

Reply:
Nukies use buzzkill box I didn’t add it in the first place